src/components/otherOperations.js

```diff
@@ -1,29 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export default class OtherOperations extends React.Component {
-  constructor(props) {
-    super(props);
-    this.handleClick = this.handleClick.bind(this);
-    this.state = {};
-  }
-
-  handleClick(e) {
-    const { handleClick } = this.props;
+const OtherOperations = (props) => {
+  const handleClick = (e) => {
+    const { handleClick } = props;
     handleClick(e.target.value);
-  }
+  };
 
-  render() {
-    return (
-      <div className="Other-Operations-container">
-        <button onClick={this.handleClick} type="button" className="Other-Operations-btn btn" value="AC">AC</button>
-        <button onClick={this.handleClick} type="button" className="Other-Operations-btn btn" value="+/-">+ / -</button>
-        <button onClick={this.handleClick} type="button" className="Other-Operations-btn btn" value="%">%</button>
-      </div>
-    );
-  }
-}
+  return (
+    <div className="Other-Operations-container">
+      <button onClick={handleClick} type="button" className="Other-Operations-btn btn" value="AC">AC</button>
+      <button onClick={handleClick} type="button" className="Other-Operations-btn btn" value="+/-">+ / -</button>
+      <button onClick={handleClick} type="button" className="Other-Operations-btn btn" value="%">%</button>
+    </div>
```

Comment:
- [ ] Please work on the functionality of `%` button currently its not working

Reply:
Hi Delice,

I tried this button, and it works as expected. Bear in mind that it's not a percentage button but a modulus operator. Its functionality was determined by the "calculate" and "operate" files provided to us by Micorverse.

Reply:
Here's a link with a video showing the modulus button working as expected: [https://drive.google.com/file/d/1XumsxbwelpLsbwPSfOMZtlbkk2ew09Xq/view?usp=sharing](https://drive.google.com/file/d/1XumsxbwelpLsbwPSfOMZtlbkk2ew09Xq/view?usp=sharing)